src/ApiService/ApiService/Functions/AgentRegistration.cs

```diff
@@ -70,9 +70,10 @@ public class AgentRegistration {
     }
 
     private async Async.Task<AgentRegistrationResponse> CreateRegistrationResponse(Service.Pool pool) {
-        var baseAddress = _context.Creds.GetInstanceUrl();
-        var eventsUrl = new Uri(baseAddress, "/api/agents/events");
-        var commandsUrl = new Uri(baseAddress, "/api/agents/commands");
+
+
+        var eventsUrl = new Uri(_context.ServiceConfiguration.OneFuzzBaseAddress, "/api/agents/events");
+        var commandsUrl = new Uri(_context.ServiceConfiguration.OneFuzzBaseAddress, "/api/agents/commands");
```

Comment:
```suggestion 
```suggestion 
    var baseAddress = _context.Creds.GetInstanceUrl();
    var eventsUrl = new Uri(baseAddress, "/api/agents/events");
    var commandsUrl = new Uri(baseAddress, "/api/agents/commands");
```
```